core/src/OpenMP/Kokkos_OpenMP_Instance.hpp

```diff
@@ -329,7 +333,7 @@ class UniqueToken<OpenMP, UniqueTokenScope::Global> {
   /// \brief acquire value such that 0 <= value < size()
   KOKKOS_INLINE_FUNCTION
   int acquire() const noexcept {
-    KOKKOS_IF_ON_HOST((return Kokkos::Impl::t_openmp_hardware_id;))
+    KOKKOS_IF_ON_HOST((return omp_get_thread_num();))
```

Comment:
we need to discuss this in a follow up, because this needs to get a global thread id not the nesting level.

Reply:
Can we add a fixme on this?